deploy/Chart/README.md

```diff
@@ -7,12 +7,19 @@ The Radius helm chart deploys the Radius services on a Kubernetes cluster using
 - Kubernetes cluster with RBAC enabled
 - Helm 3
 
-### Installing the Chart
+### Installing the Radius Chart
 
 To install the chart with the release name `radius`:
 
 ```console
 helm upgrade --wait --install radius deploy/Chart -n radius-system
+
+### Installing the Contour Chart
+
+To install the chart with the release name `contour`:
+
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm install contour bitnami/contour --namespace demo -f deploy/contour-values.yaml
```

Comment:
Just for my own understanding, could you explain why we need this? `rad install kubernetes` already installs this. Are there other scenarios where we would like to install **contour** separately? I guess on multi-tenent clusters, we have to use manual installations instead of `rad install kubernetes`.

Reply:
My organization uses a multi-tenant cluster where I only have access to a single namespace and I have no cluster scoped permissions. `rad install kubernetes` has no way to change the namespace from 'radius-system' or opt out of CRDs that need cluster-scoped permissions to define. This would not be for production but for people who want to try radius out or get buy in for approval.

Reply:
It's good that you're updating this. Also the docs for chart that are on our website are here: https://github.com/radius-project/docs/blob/v0.29/docs/content/guides/operations/kubernetes/kubernetes-install/index.md

Reply:
I think we're also open to fixing any bugs that come up when contour is not installed. We want to make it an optional component.